ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.20__amd64-nightly.yaml

```diff
@@ -4275,6 +4275,76 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-ipi-virtual-media
+- as: aws-short-cert-rotation-f9
```

Comment:
If you decided to use multi payload to run installation on arm arch nodes, you do not need to add it again in openshift-openshift-tests-private-release-4.19__arm64-nightly.yaml.